app/printManual.js

```diff
@@ -0,0 +1,18 @@
+const MajorColorNames = require('../app/majorColors');
+const MinorColorNames = require('../app/minorColors');
+const { getColorFromPairNumber } = require('../app/colorUtils');
+
+function printManual() {
```

Comment:
choose semantic name for printMannual function based on responsibility
is it Printing or Building the manual in specific  format?

Reply:
@venu-shastri Updated the file name as `colorCodeManual`